cmd/daprd/options/options.go

```diff
@@ -248,6 +251,13 @@ func New(origArgs []string) (*Options, error) {
 		opts.DaprBlockShutdownDuration = nil
 	}
 
+	if !fs.Changed("scheduler-host-address") {
```

Comment:
Since this uses StringSliceVar, the parsing of CSV values is automatic. StringSliceVar supports both CSV and repeating the flag

Reply:
This is needed for the e2e tests to not break on the n-1 and n-2 versions since `scheduler-host-address` doesn't exist in those versions.

Reply:
Not following sorry. That should be irrelevant for the n-1/n-2 tests? If it's passed or not...

Reply:
It needs to include this line due to the n-1/n-2 e2e tests not knowing the scheduler flag.